sonar-orchestrator/src/test/java/com/sonar/orchestrator/OrchestratorBuilderTest.java

```diff
@@ -235,38 +235,49 @@ public void use_default_admin_credentials_for_builds_if_enabled() {
       BuildResult successResult = mock(BuildResult.class);
       when(successResult.isSuccess()).thenReturn(true);
 
+      // Token should be created automatically
```

Comment:
These comments say what the code does, which is best to avoid in my opinion.
Instead of having multiple commented code blocks, we could extract each in a well-named method (For example `verifyTokenCreated`, `verifySecondExecutionReuseToken` etc..), . 
Thus, we avoid comments and make this test more readable.